src/components/organisms/UiMenu/stories/HasButton.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <UiPopover class="has-button">
+    <UiMenu
+      v-bind="args"
+      role="listbox"
+      :aria-label="ariaLabel"
+      aria-multiselectable="true"
+      :aria-activedescendant="ariaActivedescendant"
+      :aria-busy="ariaBusy"
+    >
+      <template #[`menu-item-${index}`] v-for="(item, index) in args.items">
+        <template v-if="item.isButton">
+          <UiButton
+            class="ui-menu-item-button__button ui-button--text"
+            @click="clickHandler"
+          >
+            <slot name="title">
+              <UiText tag="span" class="ui-menu-item-button__title">
+                <slot
+                  name="icon"
+                  v-bind="{
+                    icon,
+                    iconButtonAttrs,
+                  }"
+                >
+                  <UiIcon
+                    :icon="icon"
+                    class="ui-button__icon"
+                    v-bind="iconButtonAttrs"
+                  />
+                </slot>
+                <slot
+                  name="label"
+                  v-bind="{
+                    label,
+                    labelButtonAttrs,
+                  }"
+                >
+                  <UiText
+                    tag="span"
+                    class="ui-menu-item-button__label"
+                    v-bind="labelButtonAttrs"
+                  >
+                    {{ label }}
+                  </UiText>
+                </slot>
+              </UiText>
+            </slot>
+            <slot
+              name="hint"
+              v-bind="{
+                hint,
+                hintButtonAttrs,
+              }"
+            >
+              <UiText tag="span" class="ui-menu-item-button__hint">
+                {{ hint }}
+              </UiText>
+            </slot>
+          </UiButton>
+        </template>
+        <template v-else>
+          <UiCheckbox :aria-label="item.label" disabled />
+          <UiText tag="span">{{ item.label }}</UiText>
```

Comment:
UiCheckbox has included label :P use it instead <UiText />, probably I miss that in my template proposition